src/Bundle/test/config/sylius/grids/book_by_english_authors.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use App\Entity\Book;
+use App\Entity\Nationality;
+use Sylius\Component\Grid\Config\Builder\Field\Field;
+use Sylius\Component\Grid\Config\Builder\Filter\Filter;
+use Sylius\Component\Grid\Config\Builder\GridBuilder;
+use Sylius\Component\Grid\Config\Builder\GridConfig;
+
+return static function (GridConfig $grid) {
+    $grid->addGrid(GridBuilder::create('app_book_by_english_authors', Book::class)
+        ->setRepositoryMethod(["expr:service('app.english_books_query_builder')", 'create'])
+        ->addFilter(Filter::create('title', 'string'))
```

Comment:
All already existing filter types could be good candidates for consts. But let do this in separate PR (perhaps with predefined classes for them)